Sources/SpeziFirebaseAccountStorage/FirestoreAccountStorage.swift

```diff
@@ -107,17 +109,44 @@ public actor FirestoreAccountStorage: AccountStorageProvider {
     /// By default, the [`identifier`](https://swiftpackageindex.com/stanfordspezi/speziaccount/1.2.4/documentation/speziaccount/accountkey/identifier)
     /// provided by the account key is used as a field name.
     ///
+    /// ### Custom Encoder/Decoder Configuration
+    ///
+    /// For advanced use cases, such as integrating with libraries like PhoneNumberKit where you might want to set specific encoding/decoding strategies,
+    /// you can provide custom encoder and decoder instances with specific userInfo configurations.
+    ///
+    /// ```swift
+    /// import PhoneNumberKit
+    ///
+    /// let encoder = FirebaseFirestore.Firestore.Encoder()
+    /// encoder.userInfo[CodingUserInfoKey(rawValue: "com.roymarmelstein.PhoneNumberKit.encoding-strategy")!] = PhoneNumberEncodingStrategy.e164
+    ///
+    /// let decoder = FirebaseFirestore.Firestore.Decoder()
+    /// decoder.userInfo[CodingUserInfoKey(rawValue: "com.roymarmelstein.PhoneNumberKit.decoding-strategy")!] = PhoneNumberDecodingStrategy.e164
+    ///
+    /// let storage = FirestoreAccountStorage(
+    ///     storeIn: Firestore.firestore().collection("users"),
+    ///     encoder: encoder,
+    ///     decoder: decoder
+    /// )
+    /// ```
```

Comment:
Probably want to adjust the documentation example with the anticipation that https://github.com/marmelroy/PhoneNumberKit/pull/856 will be merged & add an example that works in the Spezi `Configuration` setting, e.g., generating the `encoders`/`decoders` inline or in computed property.